README.md

```diff
@@ -10,20 +10,24 @@ ocfweb
 
 ## Working on `ocfweb`
 
+We recommend following all of these steps
+[on supernova](https://www.ocf.berkeley.edu/docs/staff/procedures/ssh-supernova/),
+the staff login server. This will let you preview your changes.
```

Comment:
While this is true, the main reason we recommend running this on supernova is because it has the right packages, files, and network ports open through the firewall available to actually run ocfweb without extra setup, where other machines do not. I think the ability to preview changes is mostly incidental and this should be something like "We recommend following all of these steps on supernova, the staff login server, because it is already set up to run ocfweb with minimal extra setup".